docs/source/generate-schema.md

```diff
@@ -27,13 +30,17 @@ const typeDefs = `
     votes: Int
   }
 
-  # the schema allows the following query:
+  """
+  the schema allows the following query:
```

Comment:
I think this is actually meant to be a comment, not a description. One of the main benefits of the new approach in GraphQL 0.12 is that you can now have comments that don't become descriptions :]